package.json

```diff
@@ -43,7 +43,7 @@
     "mocha": "^3.2.0",
     "temp": "^0.8.1",
     "tslint": "^3.15.1",
-    "typescript": "2.2.1"
+    "typescript": "2.3.0"
```

Comment:
Note that this file also references `typescript` under "peerDependencies", but I wasn't sure if that should also be modified, since it wasn't changed when upgrading to 2.2 (it is currently set to `^2.1.0`).